### PR TITLE
Apply class Params to build commands.

### PR DESCRIPTION
**Pull request by luqp**

### CODE REVIEW

File: src/main/java/org/jalasoft/moi/model/python/PythonCommandBuilder.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+    /**
+     * Builds execution command.
+     *
+     * @param path location of the directory or file.
+     * @return execution command.
+     */
+    private String commandToRun(Path path) {
+        String fileName = path.getFileName().toString().replace(".py", "");
+        return pythonPath + SPACE + Paths.get(path.getParent() + "/__pycache__/" + fileName + ".cpython-" + version + ".pyc");
```
Comment (nanrehd) [inline src/main/java/org/jalasoft/moi/model/python/PythonCommandBuilder.java]: You may separate these calls in different lines for better debuging.

**Review by nanrehd — DISMISSED** [review]
It's a suggestion that I made, not really dangerous.
Great job, careful when solving conflicts.

File: src/main/java/org/jalasoft/moi/model/core/Executer.java [hunk 2]
```diff
@@ -16,13 +16,12 @@
 /**
  * Class recieves a string, executes it on cmd and returns output on a string
  *
- * @version 1.0 03 March 2020
  * @author Mauricio Oroza
- *
+ * @version 1.0 03 March 2020
  */
```
Comment (Snip-77) [inline src/main/java/org/jalasoft/moi/model/core/Executer.java:21]: Please flip this two lines

File: src/main/java/org/jalasoft/moi/model/core/Executer.java [hunk 3]
```diff
@@ -16,13 +16,12 @@
...
 public class Executer {
+
     private String command;
-    private String output;
```
Comment (Snip-77) [inline src/main/java/org/jalasoft/moi/model/core/Executer.java:26]: :'v

File: src/main/java/org/jalasoft/moi/model/core/ICommandBuilder.java [hunk 4]
```diff
@@ -6,22 +6,23 @@
...
 package org.jalasoft.moi.model.core;
 
 import java.nio.file.Path;
 
 /**
- * This interface defines the type of object for the classes that implement it will use
+ * Implemented for all class that want to build commands for a certain language.
  *
+ * @author Lucero Quiroga Perez
  * @version 1.0
```
Comment (Snip-77) [inline src/main/java/org/jalasoft/moi/model/core/ICommandBuilder.java:17]: Date is missing

File: src/main/java/org/jalasoft/moi/model/core/IHandler.java [hunk 5]
```diff
@@ -8,15 +8,15 @@
  */
```
Comment (Snip-77) [inline src/main/java/org/jalasoft/moi/model/core/IHandler.java:8]: Missing /** at start of the copyright

File: src/main/java/org/jalasoft/moi/model/core/IHandler.java [hunk 6]
```diff
@@ -8,15 +8,15 @@
...
 
 package org.jalasoft.moi.model.core;
+
 /**
  * Handler is in charge to work with the executer and the command builder
  *
- * @version 1.0
  * @author Carlos Meneses
+ * @version 1.0
```
Comment (Snip-77) [inline src/main/java/org/jalasoft/moi/model/core/IHandler.java:16]: Version should go first according to java conventions

File: src/main/java/org/jalasoft/moi/model/python/PythonHandler.java [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+import java.io.IOException;
+
+/**
+ * Handles the flow of the execution.
+ *
+ * @version    1.0
+ * @author     Lucero Quiroga Perez
+ */
+public class PythonHandler {
+    /**
```
Comment (Snip-77) [inline src/main/java/org/jalasoft/moi/model/python/PythonHandler.java:27]: Missing a space here

**Review by Snip-77 — COMMENTED** [review]
Please review the comments

**Review by carlosmc23 — APPROVED** [review]

**Review by nanrehd — APPROVED** [review]

**Pull request merged**